kotlin-inject-compiler-core/src/main/kotlin/me/tatarka/inject/compiler/InjectGenerator.kt

```diff
@@ -11,6 +19,7 @@ import kotlin.reflect.KClass
 private val SCOPED_COMPONENT = ClassName("me.tatarka.inject.internal", "ScopedComponent")
 private val LAZY_MAP = ClassName("me.tatarka.inject.internal", "LazyMap")
 
+@Suppress("TooManyFunctions")
```

Comment:
Should this one be disabled? I can't think of a case where I'd see it and _not_ suppress it.

Reply:
I suppose there are different schools of thought on what makes code readable and I'm not here to preach :) Disabling.